doc/nrf/software_maturity.rst

```diff
@@ -7,11 +7,25 @@ Software maturity levels
    :local:
    :depth: 2
 
-Following are the different software maturity levels shown for the different features:
+|NCS| supports its various features and components at different levels of software maturity.
+The tables at the end of this page summarize the maturity level for each feature and component supported in |NCS|.
+Three levels are used in the tables to classify the software maturity of each feature and component:
 
-* Supported - Features that are maintained and are suitable for integration in end products.
-* Experimental - Features that are suitable for prototyping or evaluation, but are not recommended to be deployed in end products.
-* Not supported - The feature or component will not work if not supported.
+Supported
+   The feature or component is maintained and is suitable for product development.
+
+Not supported
+   The feature or component does not work.
+
+Experimental
+   The feature can be used for development but it is not recommended for production.
+   This means that the feature is incomplete in functionality or verification and can be expected to change in future releases.
+   The feature is made available in its current state but the design and interfaces can change between release tags.
+   The feature is also labeled as ``EXPERIMENTAL`` in Kconfig files to indicate this status.
+
+   .. note::
+      By default, the build system generates build warnings to indicate when features labeled ``EXPERIMENTAL`` are included in builds.
+      You can disable these warnings by disabling the :kconfig:option:`CONFIG_WARN_EXPERIMENTAL` kconfig option.
```

Comment:
Can we link here? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_build_system.html#ncs-additions